test/systemtests/hostaccess_test.go

```diff
@@ -1,26 +1,44 @@
 package systemtests
 
 import (
-	"time"
-	"github.com/contiv/contivmodel/client"
 	. "github.com/contiv/check"
+	"github.com/contiv/contivmodel/client"
+	"time"
 )
 
 func (s *systemtestSuite) TestBasicHostAccess(c *C) {
 	if s.fwdMode == "routing" {
 		c.Skip("Skipping basic host access test for routing mode")
 	}
 
-	time.Sleep(30*time.Second)
+	time.Sleep(30 * time.Second)
 	global, err := s.cli.GlobalGet("global")
 	c.Assert(err, IsNil)
 	// save the FwdMode
 	fm := global.FwdMode
 	global.FwdMode = "routing"
+	if s.basicInfo.Scheduler == kubeScheduler {
+		c.Assert(s.TearDownInfraNetwork(), IsNil)
+	}
+
 	c.Assert(s.cli.GlobalPost(global), IsNil)
+
+	if s.basicInfo.Scheduler == kubeScheduler {
+		time.Sleep(60 * time.Second)
```

Comment:
GlobalPost restarts OVS on fwd-mode change and we need to wait for things to settle down before continuing